addon/components/base-chart-component.js

```diff
@@ -95,8 +95,8 @@ export default Component.extend({
                 .on('postRender', null);
         }
 
-        if (this.$() && this.$().parents() && !_.isEmpty(this.$().parents().find('.d3-tip'))) {
-            this.$().parents().find('.d3-tip').remove();
+        if (this.$() && this.$().parents() && !_.isEmpty(this.$().parents().find(`d3-tip #${this.get('elementId')}`))) {
+            this.$().parents().find(`d3-tip #${this.get('elementId')}`).remove();
```

Comment:
Doesn't this still need `.d3-tip` (preceded by period for class selection rather than no period, which is tag name)?

Reply:
Yep, you're absolutely right.